features/belongs_to.feature

```diff
@@ -27,7 +27,7 @@ Feature: Belongs To
     """
       ActiveAdmin.register User
       ActiveAdmin.register Post do
-        belongs_to :user
+        belongs_to :author, class_name: "User", param: "user_id", route_name: "user"
```

Comment:
Are these and the other changes in this file related to this PR?

Reply:
Yes, they are needed. I am not too familiar with `belongs_to`, but AFAICT the arguments are necessary for this to work at all (because the class name, `User`, does not match the attribute name, `author`). But I don't understand why the tests did not fail before. I'll try to figure out what is going on.

Reply:
Did you figure it out @c960657?